src/server/frontend_wayland/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 add_subdirectory(protocol/)
 add_subdirectory(generated/)
 
+include_directories(../frontend_xwayland)
```

Comment:
This makes me wonder if we'd be better with frontend_wayland/x11. (Just opening discussion, not suggesting a change.)